google-cloud-datastore/src/main/java/com/google/cloud/datastore/AggregationQuery.java

```diff
@@ -143,6 +144,18 @@ public Builder addAggregation(Aggregation aggregation) {
       return this;
     }
 
+    public Builder addAggregations(AggregationBuilder<?>... aggregationBuilders) {
+      for (AggregationBuilder<?> builder : aggregationBuilders) {
+        this.aggregations.add(builder.build());
+      }
+      return this;
+    }
+
+    public Builder addAggregations(Aggregation... aggregations) {
```

Comment:
maybe add one more convenience method accepting a List?

Reply:
fixed as part of 403c01c

Reply:
sorry, I meant adding something like: 

`public Builder addAggregations(List<Aggregation> aggregations)`

if you think this is not a useful convenience method, feel free to ignore :)

Reply:
sure, Let's keep and expose the minimal API for now.